Loadouts/msv_ak74_emr.hpp

```diff
@@ -0,0 +1,323 @@
+  SAM_GEAR("B_Carryall_oli", SAM_MAG)
+  launchers[] = {SAM};
+};
+class potato_msv_msamag: Fic_Spotter {// SAM Spotter/Ammo Bearer
+  SAM_GEAR("B_Carryall_oli", SAM_MAG)
+};
+class potato_msv_msaml: potato_msv_msamag {// SAM Leader
+  items[] += {LEADER_TOOLS};
+  linkedItems[] += {LEADER_LINKED,RANGE_FINDER};
+};
+class potato_msv_mtrl: Fic_Soldier_Carbine { // Mortar Leader
+  items[] += {LEADER_TOOLS};
+  linkedItems[] += {LEADER_LINKED,RANGE_FINDER};
+};
+class potato_msv_mtrg: Fic_Soldier_Carbine {// Mortar Gunner
+  MORTAR_GEAR("O_Mortar_01_weapon_F")
+};
+class potato_msv_mtrag: Fic_Spotter {// Assistant Mortar
+  MORTAR_GEAR("O_Mortar_01_support_F")
+};
+class spotter_F: Fic_Spotter {// Spotter
+  weapons[] = {SPOTTER};
+  magazines[] = {SPOTTER_MAG,BASE_GRENADES};
+  items[] += {RADIO_MR,"ACE_ATragMX","ACE_Kestrel4500","ACE_RangeCard"};
+  linkedItems[] += {LEADER_LINKED};
+  attachments[] = {"rhs_acc_pso1m2"};
+};
+class potato_msv_sniper: spotter_F {// Sniper
+  weapons[] = {SNIPER};
+  magazines[] = {SNIPER_MAG,BASE_GRENADES};
+  items[] = {BASE_TOOLS,"ACE_RangeCard"};
+  linkedItems[] = {BASE_LINKED,LEADER_LINKED};
+  attachments[] = {"rhs_acc_pso1m2"};
+};
+class potato_msv_pilot {// Pilot
+  uniform[] = {"rhs_uniform_df15"};
+  vest[] = {"V_TacVest_blk"};
+  headgear[] = {"rhs_zsh7a_mike"};
+  backpack[] = {"B_AssaultPack_rgr"};
+  weapons[] = {SMG};
+  magazines[] = {SMG_MAG,CREW_GRENADES};
+  backpackItems[] = {KEY_EAST,RADIO_LR};
+  items[] = {BASE_MEDICAL,BASE_TOOLS,LEADER_TOOLS,RADIO_MR,"ACE_NVG_Wide"}; // Theres no double tube NVGs for pilots in RHS yet
+  linkedItems[] = {BASE_LINKED,LEADER_LINKED};
+  attachments[] = {"rhs_acc_dtk"};
+};
+class potato_msv_cc: Fic_Soldier_Carbine {// Crew Chief
+  uniform[] = {"rhs_uniform_df15"};
+  vest[] = {"V_TacVest_blk"};
+  headgear[] = {"rhs_zsh7a_mike"};
+  backpack[] = {"B_AssaultPack_rgr"};
+  weapons[] = {SMG};
+  magazines[] = {SMG_MAG,CREW_GRENADES};
+  backpackItems[] = {KEY_EAST,RADIO_LR};
+  items[] = {BASE_MEDICAL,BASE_TOOLS,LEADER_TOOLS,RADIO_MR};
+  linkedItems[] = {BASE_LINKED,LEADER_LINKED};
+  attachments[] = {"rhs_acc_dtk"};
+};
+class fic_vehicle_crewman: Fic_Soldier_Carbine {// Vehicle crew base
+  headgear[] = {"rhs_tsh4","rhs_tsh4_bala","rhs_tsh4_ess","rhs_tsh4_ess_bala"};
+  magazines[] = {CARBINE_MAG,CREW_GRENADES};
+  backpackItems[] += {KEY_EAST,RADIO_LR};
+  linkedItems[] += {EAST_LEADER_LINKED,BINOS};
+  items[] += {BASE_MEDICAL};
+};
+class potato_msv_vicl: fic_vehicle_crewman {// Vehicle Leader
+  backpackItems[] = {RADIO_MR,RADIO_LR,KEY_EAST};
+  linkedItems[] = {BASE_LINKED,LEADER_LINKED};
+};
+class potato_msv_vicc: fic_vehicle_crewman {// Vehicle Crewman (non-repair)
+  backpackItems[] = {RADIO_MR,KEY_EAST};
+  linkedItems[] = {BASE_LINKED,LEADER_LINKED};
+};
+class potato_msv_vicd: fic_vehicle_crewman {// Vehicle Crewman (repair)
+  backpackItems[] = {"toolkit",KEY_EAST};
+  linkedItems[] = {BASE_LINKED,LEADER_LINKED};
```

Comment:
All classes that inherit from the fic crewman override the linkedItems with the same thing